src/google-maps/map-advanced-marker/README.md

```diff
@@ -0,0 +1,44 @@
+# MapAdvancedMarker
+
+The `MapAdvancedMarker` component wraps the [`google.maps.marker.AdvancedMarkerElement` class](https://developers.google.com/maps/documentation/javascript/reference/advanced-markers) from the Google Maps JavaScript API. The `MapAdvancedMarker` component displays a marker on the map when it is a content child of a `GoogleMap` component. Like `GoogleMap`, this component offers an `options` input as well as convenience inputs for `title`, `position`, `content`, `gmpDraggable` and `zIndex`, and supports all `google.maps.marker.AdvancedMarkerElement` events as outputs.
```

Comment:
I think we can drop the last sentence here. We shouldn't list out the individual inputs/outputs.

Reply:
Last sentence deleted.

Reply:
Apologies for splitting this into two commits. I accidentally left the outputs and the options part in the first one.